production_configs/20220524_dec_4822_tuned_TestDIffuseGammas/lstmcpipe_config_2022-05-24_TestDiffuseGammas.yaml

```diff
@@ -0,0 +1,92 @@
+# lstmcpipe generated config from PathConfigAllSkyFullDL1ab - 2022-05-24
+
+workflow_kind: lstchain
+
+# prod_id ex: local_no_n_islands. Default; v00 (if key left empty or None)
+prod_id: 24052022_dec_4822
+
+source_environment:
+  source_file: /fefs/aswg/software/conda/etc/profile.d/conda.sh
+  conda_env: lstchain-v0.9.6
+
+slurm_config:
+# dpps is the default account for lstanalyzer - other users should use aswg
+  user_account: dpps
+
+# If the image modifier settings should be calculated on the fly
+# set paths to an observed dl1 file and a simtel mc file
+# dl1_noise_tune_data_run: /fefs/aswg/data/real/DL1/20201121/v0.9.1/tailcut84/dl1_LST-1.Run02988.0000.h5
+# dl1_noise_tune_mc_run: /fefs/aswg/data/mc/DL0/20200629_prod5_trans_80/proton/zenith_20deg/south_pointing/proton_20deg_180deg_run1___cta-prod5-lapalma_4LSTs_MAGIC_desert-2158m_mono.simtel.gz
+
+lstmcpipe_version: 0.8.2
+prod_type: TestDiffuseGammas
+stages_to_run:
+- dl1ab
+- merge_dl1
+- dl1_to_dl2
+- dl2_to_irfs
+stages:
+  dl1ab:
+  - input: /fefs/aswg/data/mc/DL1/AllSky/20220527_src2_diffgamma/TestingDataset/GammaDiffuse/node_theta_14.984_az_355.158_
+    output: /fefs/aswg/data/mc/DL1/AllSky/24052022_dec_4822/TestingDataset/GammaDiffuse/node_theta_14.984_az_355.158_
```

Comment:
the output date formatting is wrong
please use `YYYYMMDD` to be consistent with all other prods